.github/workflows/maven-release.yaml

```diff
@@ -0,0 +1,88 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
+## For more information see: https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md#apache-maven-with-a-settings-path
+
+name: Maven Manual Deploy
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: "Git ref to release"
+        required: true
+      version:
+        description: "Maven version to release (without 'v' prefix)"
+        required: true
+      deployArgs:
+        description: "Additional Maven deploy arguments (e.g. '--debug -DautoReleaseAfterClose=false')"
+        required: false
```

Comment:
Going to use this to see if I can get the Maven release working, then re-release 1.1.0.